src/utils/css.utils.spec.ts

```diff
@@ -2,15 +2,15 @@ import {injectHeadCSS} from './css.utils';
 
 describe('css', () => {
   it('should inject global css in head', () => {
-    injectHeadCSS();
+    injectHeadCSS(document);
 
     let style: HTMLStyleElement | null = document.head.querySelector('style[stylo-editor]');
     expect(style).not.toBeNull();
   });
 
   it('should not inject twice global css in head', () => {
-    injectHeadCSS();
-    injectHeadCSS();
+    injectHeadCSS(document);
```

Comment:
- `npm run test` are failing
- can you also add a test for the new use case with shadow root please